From 26d29c119544faf6800f6e1056a646253c5c46da Mon Sep 17 00:00:00 2001
From: Phillip Lougher <phillip@squashfs.org.uk>
Date: Sun, 31 Aug 2014 03:31:07 +0100
Subject: [PATCH 084/120] action: add chmod() as a synonym for mode()

When testing the mode action, I'm always typing chmod() as
that's what it's called elsewhere.  So add chmod as a synonym, it
should probably have been called chmod in the first place.

Signed-off-by: Phillip Lougher <phillip@squashfs.org.uk>
---
 squashfs-tools/action.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/squashfs-tools/action.c b/squashfs-tools/action.c
index 0808cde..c442231 100644
--- a/squashfs-tools/action.c
+++ b/squashfs-tools/action.c
@@ -3141,5 +3141,6 @@ static struct action_entry action_table[] = {
 	{ "empty", EMPTY_ACTION, -2, ACTION_DIR, parse_empty_args, NULL},
 	{ "move", MOVE_ACTION, -2, ACTION_ALL_LNK, NULL, NULL},
 	{ "prune", PRUNE_ACTION, 0, ACTION_ALL_LNK, NULL, NULL},
+	{ "chmod", MODE_ACTION, -2, ACTION_ALL, parse_mode_args, mode_action },
 	{ "", 0, -1, 0, NULL, NULL}
 };
-- 
2.8.3

